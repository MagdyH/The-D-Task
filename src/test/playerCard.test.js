import React from 'react';
import renderer from 'react-test-renderer';
import PlayerCard from '../components/playerCard';


describe('PlayerCard component', () => {
    it('should render snapshot', () => {
      const component = renderer.create(<PlayerCard />);
  
      const tree = component.toJSON();
      expect(tree).toMatchSnapshot();
    });

});