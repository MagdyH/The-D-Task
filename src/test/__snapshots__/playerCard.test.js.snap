// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlayerCard component should render snapshot 1`] = `
<div
  className="row p-2 player-card"
>
  <div
    className="p-0 w-25"
  >
    <img
      alt=""
      className="rounded-circle player-img"
      src="unkown-profile-pic.jpg"
    />
  </div>
</div>
`;
